### PR TITLE
Data Foundation UI textual updates

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
https://bugzilla.redhat.com/show_bug.cgi?id=2255333

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
image
image
image
image
image
image
image

**Comment by TimothyAsirJeyasing** [comment]
image
image

File: locales/en/plugin__odf-console.json [hunk 1]
```diff
@@ -666,8 +666,8 @@
   "Deploys MultiCloud Object Gateway without block and file services.": "Deploys MultiCloud Object Gateway without block and file services.",
   "Deploys Data Foundation with block, shared fileSystem and object services.": "Deploys Data Foundation with block, shared fileSystem and object services.",
   "Deployment type": "Deployment type",
-  "Set Ceph RBD as the default StorageClass": "Set Ceph RBD as the default StorageClass",
-  "Configure a default RBD StorageClass to eliminate manual annotations within a StorageClass or selecting a specific StorageClass when making storage requests or provisions in your PVCs.": "Configure a default RBD StorageClass to eliminate manual annotations within a StorageClass or selecting a specific StorageClass when making storage requests or provisions in your PVCs.",
+  "Set Ceph RBD as the default storage class": "Set Ceph RBD as the default storage class",
```
Comment (SanjalKatiyar) [inline locales/en/plugin__odf-console.json:669]:
```suggestion
  "Set Ceph RBD as the default StorageClass": "Set Ceph RBD as the default StorageClass",
```

File: locales/en/plugin__odf-console.json [hunk 2]
```diff
@@ -666,8 +666,8 @@
...
+  "Configure default RBD storage class to avoid adding manual annotations within a storage class and selecting a specific storage class when making storage requests or provisions in your PVCs.": "Configure default RBD storage class to avoid adding manual annotations within a storage class and selecting a specific storage class when making storage requests or provisions in your PVCs.",
```
Comment (SanjalKatiyar) [inline locales/en/plugin__odf-console.json:670]:
```suggestion
  "Configure default RBD StorageClass to avoid adding manual annotations within a StorageClass and selecting a specific StorageClass when making storage requests or provisions in your PVCs.": "Configure default RBD storage class to avoid adding manual annotations within a storage class and selecting a specific storage class when making storage requests or provisions in your PVCs.",
```

File: locales/en/plugin__odf-console.json [hunk 3]
```diff
@@ -754,12 +754,11 @@
   "{{displayName}} connection details": "{{displayName}} connection details",
   "Prepare cluster for disaster recovery (Regional-DR only)": "Prepare cluster for disaster recovery (Regional-DR only)",
   "Set up the storage system for disaster recovery service with the essential configurations in place. This will subsequently allows seamless implementation of the disaster recovery strategies for your workloads.": "Set up the storage system for disaster recovery service with the essential configurations in place. This will subsequently allows seamless implementation of the disaster recovery strategies for your workloads.",
-  "No": "No",
   "Not connected": "Not connected",
   "Backing storage": "Backing storage",
   "Deployment type: {{deployment}}": "Deployment type: {{deployment}}",
   "Network file system: {{nfsStatus}}": "Network file system: {{nfsStatus}}",
-  "Set Ceph RBD as the default StorageClass: {{isCephRBDSetAsDefault}}": "Set Ceph RBD as the default StorageClass: {{isCephRBDSetAsDefault}}",
+  "Ceph RBD as the default storage class: {{isCephRBDSetAsDefault}}": "Ceph RBD as the default storage class: {{isCephRBDSetAsDefault}}",
```
Comment (SanjalKatiyar) [inline locales/en/plugin__odf-console.json:761]:
```suggestion
  "Ceph RBD as the default StorageClass: {{isCephRBDSetAsDefault}}": "Ceph RBD as the default storage class: {{isCephRBDSetAsDefault}}",
```

File: locales/en/plugin__odf-console.json [hunk 4]
```diff
@@ -1022,8 +1021,8 @@
...
+  "For enhanced performance of  the ODF cluster, the number of CPUs and memory resources are determined based on the cluster environment, size and various other factors.": "For enhanced performance of  the ODF cluster, the number of CPUs and memory resources are determined based on the cluster environment, size and various other factors.",
```
Comment (SanjalKatiyar) [inline locales/en/plugin__odf-console.json:1025]:
```suggestion
  "For enhanced performance of the ODF cluster, the number of CPUs and memory resources are determined based on the cluster environment, size and various other factors.": "For enhanced performance of  the ODF cluster, the number of CPUs and memory resources are determined based on the cluster environment, size and various other factors.",
```

Comment (SanjalKatiyar) [inline locales/en/plugin__odf-console.json:1025]:
```suggestion
  "For enhanced performance of  the ODF cluster, the number of CPUs and memory resources are determined based on the cluster environment, size and various other factors.": "For enhanced performance of  the Data Foundation cluster, the number of CPUs and memory resources are determined based on the cluster environment, size and various other factors.",
```

File: locales/en/plugin__odf-console.json [hunk 5]
```diff
@@ -1061,11 +1060,11 @@
   "Attach OBC to a Deployment": "Attach OBC to a Deployment",
   "Deployment Name": "Deployment Name",
   "Attach": "Attach",
-  "Configure monitor count": "Configure monitor count",
-  "To enhance system resilience, align Ceph monitors with the available node failure zones.": "To enhance system resilience, align Ceph monitors with the available node failure zones.",
-  "Failure domains: {{failureDomains}}": "Failure domains: {{failureDomains}}",
-  "Monitor count: {{monitorCount}}": "Monitor count: {{monitorCount}}",
-  "Recommended monitor count: 5": "Recommended monitor count: 5",
+  "Configure Ceph Monitor": "Configure Ceph Monitor",
```
Comment (SanjalKatiyar) [inline locales/en/plugin__odf-console.json:1063]:
```suggestion
  "Configure Ceph monitor": "Configure Ceph Monitor",
```

File: locales/en/plugin__odf-console.json [hunk 6]
```diff
@@ -1061,11 +1060,11 @@
...
+  "To enhance cluster resilience, align Ceph Monitors with the available node failure zones.": "To enhance cluster resilience, align Ceph Monitors with the available node failure zones.",
```
Comment (SanjalKatiyar) [inline locales/en/plugin__odf-console.json:1064]:
```suggestion
  "To enhance cluster resilience, align Ceph monitors with the available node failure zones.": "To enhance cluster resilience, align Ceph Monitors with the available node failure zones.",
```

File: locales/en/plugin__odf-console.json [hunk 7]
```diff
@@ -1061,11 +1060,11 @@
...
+  "Node failure zones: {{failureDomains}}": "Node failure zones: {{failureDomains}}",
+  "Ceph Monitor count: {{monitorCount}}": "Ceph Monitor count: {{monitorCount}}",
```
Comment (SanjalKatiyar) [inline locales/en/plugin__odf-console.json:1066]:
```suggestion
  "Ceph monitor count: {{monitorCount}}": "Ceph Monitor count: {{monitorCount}}",
```

File: locales/en/plugin__odf-console.json [hunk 8]
```diff
@@ -1061,11 +1060,11 @@
...
+  "Recommended Ceph Monitor count: 5": "Recommended Ceph Monitor count: 5",
```
Comment (SanjalKatiyar) [inline locales/en/plugin__odf-console.json:1067]:
```suggestion
  "Recommended Ceph monitor count: 5": "Recommended Ceph Monitor count: 5",
```

File: packages/odf/components/topology/Topology.tsx [hunk 9]
```diff
@@ -125,7 +125,7 @@ const MessageButton: React.FC = () => {
     <div className="odf-topology__message-button">
       <BlueInfoCircleIcon />{' '}
       {showMessage &&
-        t('This view is only supported for Internal mode cluster.')}{' '}
+        t('This view is supported only for internal mode cluster.')}{' '}
```
Comment (SanjalKatiyar) [inline packages/odf/components/topology/Topology.tsx:128]: there is another text `Topology view is not supported for External mode`, if UX do not want first letter to be capital then make it "external" here as well...

File: packages/odf/constants/tooltips.tsx [hunk 10]
```diff
@@ -72,5 +76,5 @@ export const resourceProfileTooltip = (t: TFunction) => {
 
 export const resourceRequirementsTooltip = (t: TFunction) =>
   t(
-    "plugin__odf-console~The number of CPUs and memory resources needed to optimize your Data Foundation cluster for enhanced performance is determined by taking into account the cluster's specific environment, size and various other factors."
+    'plugin__odf-console~For enhanced performance of  the ODF cluster, the number of CPUs and memory resources are determined based on the cluster environment, size and various other factors.'
```
Comment (SanjalKatiyar) [inline packages/odf/constants/tooltips.tsx:79]:
```suggestion
    'plugin__odf-console~For enhanced performance of  the Data Foundation cluster, the number of CPUs and memory resources are determined based on the cluster environment, size and various other factors.'
```

File: plugins/odf/console-extensions.json [hunk 11]
```diff
@@ -669,7 +669,7 @@
   {
     "type": "console.alert-action",
     "properties": {
-      "alert": "CephMonLowNumber",
+      "alert": "Inadequate Ceph Monitor count",
```
Comment (SanjalKatiyar) [inline plugins/odf/console-extensions.json:672]: why are you changing this ?? please make sure you know why this change is needed/not needed...

File: packages/odf/types/common.ts [hunk 12]
```diff
@@ -40,9 +40,9 @@ export type NodesPerZoneMap = {
 };
 
 export enum ResourceProfile {
-  Lean = 'lean', // t('Lean')
-  Balanced = 'balanced', // t('Balanced')
-  Performance = 'performance', // t('Performance')
+  Lean = 'Lean', // t('Lean')
+  Balanced = 'Balanced', // t('Balanced')
+  Performance = 'Performance', // t('Performance')
```
Comment (SanjalKatiyar) [inline packages/odf/types/common.ts:45]: do you need to make this change ?? unnecessary changes can cause unwanted bug, enum `ResourceProfile` is used at multiple places...

File: packages/odf/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/configure-performance.tsx [hunk 13]
```diff
@@ -29,11 +29,8 @@ import './configure-performance.scss';
 
 const selectOptions = (t: TFunction, forceLean: boolean) =>
   Object.entries(ResourceProfile).map((value: [string, ResourceProfile]) => {
-    let displayName = value[0];
+    const displayName = t(`${value[0]} mode`);
     let profile = value[1];
-    if (profile === ResourceProfile.Balanced) {
-      displayName = t(`${displayName} (default)`);
-    }
```
Comment (SanjalKatiyar) [inline packages/odf/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/configure-performance.tsx:36]: @Chandan141996 earlier we added an item to the Performance mode dropdown as `Balanced mode (default)`, do we need to remove this `default` part now ??

**Comment by SanjalKatiyar** [comment]
do not use term "ODF" or "Openshift" anywhere... use "Data Foundation" instead...

**Comment by SanjalKatiyar** [comment]
add BZ link to PR description...

**Comment by TimothyAsirJeyasing** [comment]
@GowthamShanmugam , @SanjalKatiyar Please review

File: locales/en/plugin__odf-console.json (hunk 1 above)
Comment (Chandan141996) [reply locales/en/plugin__odf-console.json:669]: @TimothyAsirJeyasing @SanjalKatiyar I updated the text against the checkbox here to 'Use Ceph RBD as the default storage class', please update it. Use the mockup for reference

**Comment by GowthamShanmugam** [comment]
> @GowthamShanmugam , @SanjalKatiyar Please review

Review is already done, Please resolve the review comments

File: packages/odf/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/configure-performance.tsx (hunk 13 above)
Comment (Chandan141996) [reply packages/odf/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/configure-performance.tsx:36]: Yes, I don't think we need to mention 'default', as this is going to be the default selection when there are ample resources, in case when there are low resources 'lean mode' is going to be the default selection for users. I don't see a reason to highlight this, unless we have a really good reason to. If you have other suggestions, please let me know.

Comment (SanjalKatiyar) [reply packages/odf/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/configure-performance.tsx:36]: ack ! just wanted to confirm if it's intentional or a miss !!

File: plugins/odf/console-extensions.json (hunk 11 above)
Comment (Chandan141996) [reply plugins/odf/console-extensions.json:672]: @SanjalKatiyar This change was suggested by microcopy for the alert message in the status card, would changing the title also going to impact or have changes to the original alert??

**Comment by Chandan141996** [comment]
Would be helpful to see the screenshots for the 'Configure cephMon' epic too @TimothyAsirJeyasing @SanjalKatiyar

File: plugins/odf/console-extensions.json (hunk 11 above)
Comment (SanjalKatiyar) [reply plugins/odf/console-extensions.json:672]: `CephMonLowNumber` is the name of the Alert that comes from Prometheus and have its own naming convention. Making above change will break the flow and no Alert will popup on the dashboard (due of mismatch). However, Alerts have description, that is something we can update (changes needs to be done on the operator side which add rules for this).
Also this is "actionable" Alert which opens a modal, so there we can make these changes and display any text we want.

File: packages/odf/components/utils/common-odf-install-el.tsx [hunk 14]
```diff
@@ -42,23 +42,21 @@ export const VALIDATIONS = (
...
     case ValidationType.RESOURCE_PROFILE:
       return {
         variant: AlertVariant.danger,
-        title: t(
-          'Aggregate resource requirements for the selected performance profile not met.'
-        ),
+        title: t('Aggregate resource requirement for performance mode not met'),
         text: t(
-          'Select nodes with sufficient CPU and memory that meet the specified minimum requirements, and try again, or choose a different performance profile to proceed.'
+          'The selected nodes do not meet the performance mode aggregate resource requirement. Try again by selecting nodes with enough CPU and memory or you can select a different performance profile to proceed.'
```
Comment (SanjalKatiyar) [inline packages/odf/components/utils/common-odf-install-el.tsx:59]: this is wrong, older/newer texts have entirely different meanings...
earlier it was `performance profile` which could have been any Lean/Balanced/Performance, but newer updated text always says `performance mode` which is wrong and should be dynamic based upon selected profile...

File: packages/odf/components/utils/common-odf-install-el.tsx [hunk 15]
```diff
@@ -42,23 +42,21 @@ export const VALIDATIONS = (
...
         actionLinkStep: CreateStepsSC.STORAGEANDNODES,
         actionLinkText: t('Back to nodes selection'),
       };
     case ValidationType.RESOURCE_PROFILE:
       return {
         variant: AlertVariant.danger,
-        title: t(
-          'Aggregate resource requirements for the selected performance profile not met.'
-        ),
+        title: t('Aggregate resource requirement for performance mode not met'),
```
Comment (SanjalKatiyar) [inline packages/odf/components/utils/common-odf-install-el.tsx:57]: same here, u can't always use `performance mode`, this should be dynamic based upon selected profile...

File: plugins/odf/console-extensions.json (hunk 11 above)
Comment (TimothyAsirJeyasing) [reply plugins/odf/console-extensions.json:672]: i will use CephMonLowNumber for now, later once its ready i will send a new updated pr

File: packages/odf/components/topology/Topology.tsx (hunk 9 above)
Comment (TimothyAsirJeyasing) [reply packages/odf/components/topology/Topology.tsx:128]: Checked with UX, and i will keep the same case as used before. Will update later in a separate patch if required.

File: locales/en/plugin__odf-console.json (hunk 1 above)
Comment (TimothyAsirJeyasing) [reply locales/en/plugin__odf-console.json:669]: done

**Comment by TimothyAsirJeyasing** [comment]
image

File: packages/odf/components/topology/Topology.tsx (hunk 9 above)
Comment (SanjalKatiyar) [reply packages/odf/components/topology/Topology.tsx:128]: why in different PR ? it is one line change, or it is not confirmed what case to use yet ??

**Comment by TimothyAsirJeyasing** [comment]
image
image

File: packages/odf/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/configure-performance.tsx [hunk 16]
```diff
@@ -29,11 +29,8 @@ import './configure-performance.scss';
 
 const selectOptions = (t: TFunction, forceLean: boolean) =>
   Object.entries(ResourceProfile).map((value: [string, ResourceProfile]) => {
-    let displayName = value[0];
+    const displayName = t(`${value[0]} mode`);
```
Comment (GowthamShanmugam) [inline packages/odf/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/configure-performance.tsx:32]:
```suggestion
    const displayName = t("{{profileType}} mode", {profileType: value[0]});
```

File: packages/odf/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/configure-performance.tsx [hunk 17]
```diff
@@ -161,7 +158,7 @@ const ConfigurePerformance: React.FC<ConfigurePerformanceProps> = ({
           className="pf-u-font-size-sm pf-u-disabled-color-100"
         >
           {t(
-            "Customize your Data Foundation cluster's performance by selecting a performance profile that meets your specific needs."
+            'Select a profile to customise the performance of the Data Foundation cluster to meet your requirements.'
```
Comment (GowthamShanmugam) [inline packages/odf/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/configure-performance.tsx:161]: is it customize or customise?

File: packages/odf/components/utils/common-odf-install-el.tsx [hunk 18]
```diff
@@ -42,11 +43,11 @@ export const VALIDATIONS = (
         variant: AlertVariant.warning,
         title: (
           <div className="ceph-minimal-deployment-alert__header">
-            {t('A minimal cluster deployment will be performed.')}
+            {t('Deploy a lean mode cluster?')}
```
Comment (GowthamShanmugam) [inline packages/odf/components/utils/common-odf-install-el.tsx:46]: this not sound like a question

File: packages/odf/components/utils/common-odf-install-el.tsx [hunk 19]
```diff
@@ -55,10 +56,10 @@ export const VALIDATIONS = (
       return {
         variant: AlertVariant.danger,
         title: t(
-          'Aggregate resource requirements for the selected performance profile not met.'
+          `Aggregate resource requirement for ${resourceProfile} mode not met`
         ),
         text: t(
-          'Select nodes with sufficient CPU and memory that meet the specified minimum requirements, and try again, or choose a different performance profile to proceed.'
+          `The selected nodes do not meet the ${resourceProfile} mode aggregate resource requirement. Try again by selecting nodes with enough CPU and memory or you can select a different performance profile to proceed.`
```
Comment (GowthamShanmugam) [inline packages/odf/components/utils/common-odf-install-el.tsx:62]:
```suggestion
          'The selected nodes do not meet the {{resourceProfile}} mode aggregate resource requirement. Try again by selecting nodes with enough CPU and memory or you can select a different performance profile to proceed.', {resourceProfile}
```

File: locales/en/plugin__odf-console.json [hunk 20]
```diff
@@ -494,7 +494,7 @@
   "Replica {{poolSize}} {{compressionText}}": "Replica {{poolSize}} {{compressionText}}",
   "Create New Pool": "Create New Pool",
   "Storage system": "Storage system",
-  "StorageSystem which will be used for storage needs": "StorageSystem which will be used for storage needs",
+  "Select a storage system for your workloads.": "Select a storage system for your workloads.",
```
Comment (SanjalKatiyar) [inline locales/en/plugin__odf-console.json:497]:
```suggestion
  "Select a StorageSystem for your workloads.": "Select a storage system for your workloads.",
```

File: packages/odf/components/utils/common-odf-install-el.tsx (hunk 18 above)
Comment (Chandan141996) [reply packages/odf/components/utils/common-odf-install-el.tsx:46]: Disha in her review asked to frame it as a question since the user has the choice to either proceed with this deployment or select another mode by adding resources, so it's okay.

File: packages/odf/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/configure-performance.tsx (hunk 17 above)
Comment (Chandan141996) [reply packages/odf/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/configure-performance.tsx:161]: Both are correct, tbh. In the US it's spelled as 'Customize' elsewhere it's 'Customise', we can use 'Customize' here for US English speakers.

Comment (TimothyAsirJeyasing) [reply packages/odf/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/configure-performance.tsx:161]: As per UX microcopy its customise only

File: packages/odf/components/utils/common-odf-install-el.tsx (hunk 18 above)
Comment (TimothyAsirJeyasing) [reply packages/odf/components/utils/common-odf-install-el.tsx:46]: @Chandan141996 ^^

File: locales/en/plugin__odf-console.json [hunk 21]
```diff
@@ -666,8 +666,8 @@
...
   "Deploys Data Foundation with block, shared fileSystem and object services.": "Deploys Data Foundation with block, shared fileSystem and object services.",
   "Deployment type": "Deployment type",
-  "Set Ceph RBD as the default StorageClass": "Set Ceph RBD as the default StorageClass",
-  "Configure a default RBD StorageClass to eliminate manual annotations within a StorageClass or selecting a specific StorageClass when making storage requests or provisions in your PVCs.": "Configure a default RBD StorageClass to eliminate manual annotations within a StorageClass or selecting a specific StorageClass when making storage requests or provisions in your PVCs.",
+  "Use Ceph RBD as the default StorageClass": "Use Ceph RBD as the default StorageClass",
+  "Configure default RBD StorageClass to avoid adding manual annotations within a StorageClass and selecting a specific StorageClass when making storage requests or provisions in your PVCs.": "Configure default RBD StorageClass to avoid adding manual annotations within a StorageClass and selecting a specific StorageClass when making storage requests or provisions in your PVCs.",
```
Comment (SanjalKatiyar) [inline locales/en/plugin__odf-console.json:670]:
```suggestion
  "Configure default RBD StorageClass to avoid adding manual annotations within a StorageClass and selecting a specific StorageClass when making storage requests or provisions in your PVCs.": "Configure default RBD StorageClass to avoid adding manual annotations within a StorageClass or selecting a specific StorageClass when making storage requests or provisions in your PVCs.",
```
@Chandan141996 it should be `or`. Also, can we remove `selecting a specific StorageClass when making storage requests or provisions in your PVCs.` part entirely ??

Comment (SanjalKatiyar) [inline locales/en/plugin__odf-console.json:670]:
```suggestion
  "Configure default RBD StorageClass to avoid adding manual annotations within a StorageClass and selecting a specific StorageClass when making storage requests or provisions in your PVCs.": "Configure default RBD StorageClass to avoid adding manual annotations within a StorageClass or selecting a specific StorageClass when making storage requests or provisions in your PVCs.",
```
@Chandan141996 it should be `or`.
Also, can we remove `selecting a specific StorageClass when making storage requests or provisions in your PVCs.` part entirely ??

Comment (Chandan141996) [reply locales/en/plugin__odf-console.json:670]: The same or similar was mentioned in the epic, would advise not making any last-minute changes unless its important or has high consequences of user experience @SanjalKatiyar

File: packages/odf/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/capacity-and-nodes-step.tsx [hunk 22]
```diff
@@ -506,7 +506,11 @@ export const CapacityAndNodes: React.FC<CapacityAndNodesProps> = ({
       {!!validations.length &&
         !!capacity &&
         validations.map((validation) => (
-          <ValidationMessage key={validation} validation={validation} />
+          <ValidationMessage
+            resourceProfile={resourceProfile}
+            key={validation}
+            validation={validation}
+          />
```
[the remark this answers is not in the document]
Comment (Chandan141996) [reply packages/odf/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/capacity-and-nodes-step.tsx:513]: Agreed!!

File: locales/en/plugin__odf-console.json (hunk 21 above)
Comment (SanjalKatiyar) [reply locales/en/plugin__odf-console.json:670]: it definitely won't have high impact !!
so if this is something which needs to be gone through doc team first and will probably take a while, then it is okay to ignore and do not block this PR... otherwise, good to make it `or`...

File: packages/odf/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/capacity-and-nodes-step.tsx (hunk 22 above)
[the remark this answers is not in the document]
Comment (TimothyAsirJeyasing) [reply packages/odf/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/capacity-and-nodes-step.tsx:513]: Yes agreed, did't notice, Shall i do it as a bug fix later

[the remark this answers is not in the document]
Comment (SanjalKatiyar) [reply packages/odf/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/capacity-and-nodes-step.tsx:513]: this is a bug fix, and it needs to be fixed as part of this PR...

[the remark this answers is not in the document]
Comment (TimothyAsirJeyasing) [reply packages/odf/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/capacity-and-nodes-step.tsx:513]: Ok, Sure, will update the PR

File: locales/en/plugin__odf-console.json (hunk 21 above)
Comment (Chandan141996) [reply locales/en/plugin__odf-console.json:670]: Yeah, I remember now, Disha changed it since these are two separate actions, that the user can perform in two different places in the UI. Hence replaced the 'or' with an 'and' in the original draft of the message.

File: packages/odf/modals/configure-mons/configure-mons.tsx [hunk 23]
```diff
@@ -104,7 +104,7 @@ const LowMonAlertModal: ModalComponent = ({ closeModal }) => {
         >
           <FlexItem>
             {t(
-              'To enhance system resilience, align Ceph monitors with the available node failure zones.'
+              'To enhance cluster resilience, align Ceph Monitors with the available node failure zones.'
```
Comment (SanjalKatiyar) [inline packages/odf/modals/configure-mons/configure-mons.tsx:107]: @bipuladh is `failure zones` correct word here ?? or should it be `failure domains` ??

File: packages/odf/modals/configure-mons/configure-mons.tsx [hunk 24]
```diff
@@ -115,15 +115,17 @@ const LowMonAlertModal: ModalComponent = ({ closeModal }) => {
               {!nodesLoadError && (
                 <FlexItem>
                   {nodesLoaded ? (
-                    t('Failure domains: {{failureDomains}}', { failureDomains })
+                    t('Node failure zones: {{failureDomains}}', {
```
Comment (SanjalKatiyar) [inline packages/odf/modals/configure-mons/configure-mons.tsx:118]: same... @bipuladh is `failure zones` correct word here ?? or should it be `failure domains` ??

File: packages/odf/modals/configure-mons/configure-mons.tsx (hunk 23 above)
Comment (bipuladh) [reply packages/odf/modals/configure-mons/configure-mons.tsx:107]: domains

File: packages/odf/modals/configure-mons/configure-mons.tsx (hunk 24 above)
Comment (bipuladh) [reply packages/odf/modals/configure-mons/configure-mons.tsx:118]: domains

File: packages/odf/modals/configure-mons/configure-mons.tsx (hunk 23 above)
Comment (SanjalKatiyar) [reply packages/odf/modals/configure-mons/configure-mons.tsx:107]: fyi @Chandan141996
no need to talk to doc team or update the mocks... it is a minor change, we can directly update the PR...

File: packages/odf/modals/configure-mons/configure-mons.tsx (hunk 24 above)
Comment (TimothyAsirJeyasing) [reply packages/odf/modals/configure-mons/configure-mons.tsx:118]: ok i will change to "failure domains"

Comment (TimothyAsirJeyasing) [reply packages/odf/modals/configure-mons/configure-mons.tsx]: done
